### PR TITLE
Improved touch support

### PR DESCRIPTION
**Pull request by IchabodFletchman**

### CODE REVIEW

File: xml/DialogAddonInfo.xml [hunk 1]
```diff
@@ -271,5 +271,6 @@
 				<param name="sub_label" value="$INFO[ListItem.AddonVersion,[COLOR grey],[/COLOR]]$INFO[ListItem.AddonCreator, $LOCALIZE[31071] ]" />
 			</include>
 		</control>
+		<include>TouchBackButton</include>
```
Comment (phil65) [inline xml/DialogAddonInfo.xml]: please add condition="Skin.HasSetting(touchmode") for all these includes.

Comment (IchabodFletchman) [inline xml/DialogAddonInfo.xml]: I've already added that condition to the TouchBackButton include itself.
The buttons are only visible when touch mode is on (working correctly), but I can change it to your method if you prefer?

File: xml/Includes.xml [hunk 2]
```diff
@@ -1415,7 +1415,6 @@
 			<align>center</align>
 			<aligny>center</aligny>
 			<onclick>Action(Back)</onclick>
-			<visible>Skin.HasSetting(touchmode) + [!Window.IsActive(home) | $EXP[infodialog_active]]</visible>
```
Comment (phil65) [inline xml/Includes.xml]: $EXP[infodialog_active] isnt needed anymore?

Comment (IchabodFletchman) [inline xml/Includes.xml]: No. I've changed the vis conditions for the individual info dialogs like you suggested. And also added the IsActive(home) to the vis condition on the top bar.
So it isn't necessary to make that check anymore.

File: xml/View_50_List.xml [hunk 3]
```diff
@@ -30,6 +30,7 @@
 						<onright>50</onright>
 						<orientation>vertical</orientation>
 						<animation effect="zoom" end="50,100" time="300" tween="sine" center="20,0" easing="inout" condition="!Control.HasFocus(50600)">conditional</animation>
+						<hitrect x="-30" y="0" w="120" h="3000" />
```
Comment (phil65) [inline xml/View_50_List.xml]: this hitrect looks weird?

**Pull request merged**